<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>588</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackMain">
      <widget class="QWidget" name="pgAuth">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="ledtLogin">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="placeholderText">
           <string>Пароль преподавателя...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="btnSuper">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>ПРЕПОДАВАТЕЛЬ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="btnStudent">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>СТУДЕНТ</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>321</width>
            <height>237</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" colspan="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>321</width>
            <height>237</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0" rowspan="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>223</width>
            <height>81</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3" rowspan="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>223</width>
            <height>81</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgProfessor">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="1">
         <widget class="QPushButton" name="btnProDelLesson">
          <property name="text">
           <string>УДАЛИТЬ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="3">
         <widget class="QTextBrowser" name="brProShow"/>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="btnProAddLesson">
          <property name="text">
           <string>ДОБАВИТЬ</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="btnProAddExample">
          <property name="text">
           <string>ДОБАВИТЬ</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="btnProDelExample">
          <property name="text">
           <string>УДАЛИТЬ</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QScrollArea" name="scrlProLessons">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="areaProLessons">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>146</width>
             <height>476</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="layProLessons"/>
          </widget>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QScrollArea" name="scrlProExamples">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="areaProExamples">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>146</width>
             <height>476</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="layProExamples"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgStudent">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackStudent">
          <widget class="QWidget" name="pgLessons">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QScrollArea" name="scrlLessons">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="areaLessons">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>129</width>
                 <height>476</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="layLessons"/>
              </widget>
             </widget>
            </item>
            <item row="0" column="1" rowspan="3">
             <widget class="QTextBrowser" name="brLessonInfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnOpenExamples">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>ПРАКТИКА</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnExit">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>ВЫХОД</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="pgExamples">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QScrollArea" name="scrlExamples">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="lineWidth">
               <number>1</number>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="areaExamples">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>101</width>
                 <height>476</height>
                </rect>
               </property>
               <layout class="QVBoxLayout" name="layExamples"/>
              </widget>
             </widget>
            </item>
            <item row="0" column="1" rowspan="3">
             <widget class="QTextBrowser" name="brExampleInfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="btnOpenLessons">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>ТЕОРИЯ</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="btnExit_2">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>ВЫХОД</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
